lib/services/local/presence.go

```diff
@@ -1553,6 +1553,32 @@ func (s *PresenceService) DeleteAllWindowsDesktopServices(ctx context.Context) e
 	return s.DeleteRange(ctx, startKey, backend.RangeEnd(startKey))
 }
 
+// UpsertLocalClusterName upserts local cluster name
+func (s *PresenceService) UpsertUserLoginTime(ctx context.Context, name string, loginTime time.Time) error {
```

Comment:
```suggestion
// UpsertUserLoginTime upserts a host user's login time.
func (s *PresenceService) UpsertUserLoginTime(ctx context.Context, name string, loginTime time.Time) error {
```

Even then, the name is perhaps a bit improper because we use this to record the beginning of non-interactive executions too, which traditionally are not "logins" (they're not stored in `utmp`, for instance), but I don't really have better ideas. Perhaps `UserInteractionTime`?

Reply:
We should probably also mention that this is for host (i.e. unix) users, so we don't confuse ourselves thinking this is related to teleport users at all.